PZ/WebClient/models.py

```diff
@@ -6,3 +6,140 @@ class Monitors(models.Model):
     monitor_name = models.CharField(max_length=200)
     monitor_domain = models.URLField()
     user_id = models.ForeignKey(User)
+
+
+class Page(object):
+	def __init__(self, size, number, totalCount):
+		# self.dict = json.loads(json_obj)
+		self.size = size

```

Comment:
Make this members not public by adding underscore before names, like:
`self._size = size`